staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/structuredmerge.go

```diff
@@ -55,15 +57,16 @@ func NewStructuredMergeManager(models openapiproto.Models, objectConverter runti
 		groupVersion:    gv,
 		hubVersion:      hub,
 		updater: merge.Updater{
-			Converter: internal.NewVersionConverter(typeConverter, objectConverter, hub),
+			Converter:     internal.NewVersionConverter(typeConverter, objectConverter, hub),
+			IgnoredFields: resetFields,
```

Comment:
What do you think about the discrepancy between the names? Do we ignore the fields or do we reset them? ;-)

Reply:
Both names make sense at their respective places. Update and therefore the fieldManager really ignore them, but the strategies reset or wipe them.
I'm not sure if ignoredFields would fit in the strategy/storage context as the logic does not ignore but overwrite the users intention.